ci/setup_env.sh

```diff
@@ -218,41 +225,48 @@ kubectl get pvc,pv
 
 # Create the application
 echo "Creating busybox-jiva and busybox-cstor application pod"
-kubectl create -f $GOPATH/src/github.com/openebs/maya/ci/snapshot/jiva/busybox.yaml
-kubectl create -f $GOPATH/src/github.com/openebs/maya/ci/snapshot/cstor/busybox.yaml
+kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/jiva/busybox.yaml
+kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/cstor/busybox.yaml
+kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/cstor/busybox_ns.yaml
 
 for i in $(seq 1 100) ; do
     phaseJiva=$(kubectl get pods busybox-jiva --output="jsonpath={.status.phase}")
     phaseCstor=$(kubectl get pods busybox-cstor --output="jsonpath={.status.phase}")
-    if [ "$phaseJiva" == "Running" ] && [ "$phaseCstor" == "Running" ]; then
+    phaseCstorNs=$(kubectl get pods busybox-cstor-ns --output="jsonpath={.status.phase}")
+    if [ "$phaseJiva" == "Running" ] && [ "$phaseCstor" == "Running" ] && [ "$phaseCstorNs" == "Running" ]; then
         break
 	else
         echo "busybox-jiva pod is in:" $phaseJiva
         echo "busybox-cstor pod is in:" $phaseCstor
+        echo "busybox-cstor-ns pod is in:" $phaseCstorNs
 
         if [ "$phaseJiva" != "Running" ]; then
            kubectl describe pods busybox-jiva
         fi
         if [ "$phaseCstor" != "Running" ]; then
            kubectl describe pods busybox-cstor
         fi
+        if [ "$phaseCstorNs" != "Running" ]; then
+           kubectl describe pods busybox-cstor-ns
+        fi
         sleep 10
     fi
 done
 
 dumpMayaAPIServerLogs 100
 
 echo "********************Creating volume snapshot*****************************"
-kubectl create -f  $GOPATH/src/github.com/openebs/maya/ci/snapshot/jiva/snapshot.yaml
-kubectl create -f  $GOPATH/src/github.com/openebs/maya/ci/snapshot/cstor/snapshot.yaml
+kubectl create -f  https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/jiva/snapshot.yaml
+kubectl create -f  https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/cstor/snapshot.yaml
+kubectl create -f  https://raw.githubusercontent.com/openebs/openebs/master/ci/snapshot/cstor/snapshot_ns.yaml
```

Comment:
nit: link will be https://raw.githubusercontent.com/openebs/openebs/master/ci/maya/snapshot/cstor/snapshot_ns.yaml ?

Reply:
see the `maya` is missing in between

Reply:
Thanks a lot!